protocol/synthetix/contracts/storage/VaultEpoch.sol

```diff
@@ -64,12 +64,7 @@ library VaultEpoch {
          * and directly when users mint or burn USD, or repay debt.
          */
         mapping(uint256 => int256) consolidatedDebtAmountsD18;
-        /**
-         * @dev Tracks last time a user delegated to this vault.
-         *
-         * Needed to validate min delegation time compliance to prevent small scale debt pool frontrunning
-         */
-        mapping(uint128 => uint64) lastDelegationTime;
+        mapping(uint128 => uint64) __unused_legacy_slot;
```

Comment:
Should be marked as `do not use`, because maybe in the future someone reads "unused" and think it has no data.

Reply:
Ah! For consistency it should be __camelCase